Réécriture totale du jeu

SomeBoringNerd opened the pull request:
>Le jeu marche comme avant
>Textures en meilleure qualité
>Github Actions pour distribution automatique
>Support de la manette
>Musiques + SFXs

ps : me dispute pas c'est pas mon idée c'est @kaissouDev qui m'a dit de le faire :<

ObviousAltAccount committed:
[compile pas] Premier commit

ObviousAltAccount committed:
[compile pas] Update gitignore

ObviousAltAccount committed:
[compile]
>Update des textures
>Début de la réécriture du joueur
>Fix d'un bug avec le système d'inputs

ObviousAltAccount committed:
[compile]
>Réécriture du code joueur
>Début de la recréation du code de génération de la map

ObviousAltAccount committed:
[compile]
>Réécriture complète du jeu pour utiliser Transcendance

ObviousAltAccount committed:
[compile]
>Github action

ObviousAltAccount committed:
[compile]
>Github action

ObviousAltAccount committed:
[compile]
>Github action essais numéro 3

ObviousAltAccount committed:
[compile]
>Github action essais numéro 4

ObviousAltAccount committed:
[compile]
>Système de génération de map fonctionnel
>Système de chunk threadé

ObviousAltAccount committed:
[compile]
>Ajout d'arbres, et de structures
>Optimisations diverses des collisions
>Fix d'un lagspike lié a la création d'un chunk
* : la génération des blocks est désormais complètement threadée
>Fix de problèmes de physique lié au joueur
>Ajout de la possibilité de casser des blocks
>Ajout d'un écran de crédits
>Ajout d'une musique custom

ObviousAltAccount committed:
Oops, j'ai mal reglé les buildfiles

ObviousAltAccount committed:
Ne jamais utiliser la commande "clear" dans github action, pire erreur de ma vie

ObviousAltAccount committed:
La putain de toi tu va compiler fils de pute ?

ObviousAltAccount committed:
[compile]
Par pitié compile je t'en supplie

ObviousAltAccount committed:
[compile]
>Réécriture du système de chunks, de structure et de cassage de blocks
>Ajout du block de bedrock
>encore plus d'optimisation de collisions
>ajout de sfx pour les boutons, sauter et casser des blocks
>ajout d'un curseur pour cacher le fait que la souris n'est pas sync avec avec le jeu mdr
>ajout d'une box outline pour faciliter le fait de voir quel block est selectionné

ObviousAltAccount committed:
[compile]
>Ajout d'un système de blocks
>les blocks de grass bloqués se transforment en dirt désormais
>les blocks de dirts a coté de grass se transforment en grass désormais
>nettoyage de l'engine pour enlever du code innutile

ghost commented:
note : je recommande pas de merge perso, le but c'est que les gens forkent le projet original, pas le mien mdr

ObviousAltAccount committed:
[compile]
>Documentation du code
>Revamp de la gestion de la déclaration des blocks
>début de l'ajout de l'inventaire ainsi que du système d'items

ObviousAltAccount committed:
[compile] Maman j'ai encore cassé github action

ObviousAltAccount committed:
[compile] Ajout d'une nouvelle musique et retrait de quelques typos

ObviousAltAccount committed:
[compile] Ajout de la hotbar
>Revamp du système de poser / casser des blocks
>Les blocks vont désormais se dropper
>Ajout du stick
>Augmentation du volume de la musique de 150% (de 0.2f a 0.5f)

ObviousAltAccount committed:
[compile] Ajout de la hotbar
>Revamp du système de poser / casser des blocks
>Les blocks vont désormais se dropper
>Ajout du stick
>Augmentation du volume de la musique de 150% (de 0.2f a 0.5f)
>Ajout de DroppedItemEntity qui n'a pas été ajouté a git je suis trop conne bordel

ObviousAltAccount committed:
[compile] Ajout de la hotbar
>Le système d'inventaire a été ajouté
>Réécriture partielle du système de dropped items, de la hotbar et de l'inventory manager

ObviousAltAccount committed:
[compile] Dernier commit pour le moment
